crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -743,6 +743,18 @@ en:
       non_json_file: 'Unprocessable chef file found'
       invalid_file_extension: 'Filename needs to be a valid tarball with a .tar.gz file extension'
 
+  sanities:
+    show:
+      header: 'Sanity Check Failures'
+      notice: |
+        There are errors that must be resolved. Please restart Crowbar after solving the errors.
+      fqdn_detected: 'Unable to detect fully-qualified hostname.'
+      ip_resolved: 'Could not resolve %{fqdn} to an IPv4 or IPv6 address.'
+      loopback_unresolved: '%{fqdn} resolves to a loopback address.'
+      ip_configured: 'No local interface is configured with an correct ip address.'
```

Comment:
s/an/a/ (I think) and s/ip/IP/

Would it be possible to specify what is a correct IP address? :-)